conf/ephys_upload_job_configs.yml

```
@@ -0,0 +1,27 @@
+# Leave values blank to use defaults
+raw_data:
+  name: openephys
+  source_dir: /home/jon.young/Projects/nd-data-transfer/tests/resources/v0.6.x_neuropixels_multiexp_multistream
+clip_data_job:
+  clip: true # Set as false to skip clip step
+  clipped_data_dest: /home/jon.young/Projects/nd-data-transfer/tests/resources/ephys_clipped_data
```

```suggestion
  clipped_data_dest: /home/jon.young/Projects/nd-data-transfer/tests/resources/ecephys_clipped
```
Let's keep the same nomenclature